src/main/resources/swagger/api-docs.yaml

```diff
@@ -3191,6 +3220,25 @@ components:
           type: boolean
           description: Use the pet service account project as the user project in the request.
           default: true
+    RequesterPaysSignedUrlRequest:
+      type: object
+      required:
+        - bucketName
+        - blobName
+      properties:
+        bucketName:
```

Comment:
I know this is copying existing form but I always found it weird that this was two variables rather than a gs path

Reply:
I'm down to make it just take in a GS Path. I'll do that!